javascript.js

```diff
@@ -0,0 +1,20 @@
+// set variavles
+var user = []; //inline coment
+
+/**
+ * adds two numbers togeter
+ * 
+ * @param {number} first the first number
+ * @returns {number}
+ */
+function add(first) {
+  /*
```

Comment:
Typo inside comment in word `ere`. Possible corrections:  `pre` (27), `re` (11), `were` (3)
Typo inside comment in word `coment`. Possible corrections:  `comment` (59), `moment` (40), `cement` (38)